src/codecs.ts

```diff
@@ -871,3 +873,61 @@ export const CookieCsvInput = t.intersection([
 
 /** Type override */
 export type CookieCsvInput = t.TypeOf<typeof CookieCsvInput>;
+
+/// //////////////////////////////////////
+// Transcend Proxy AI Server policies   //
+/// //////////////////////////////////////
+export const Policies = t.literal('redactEmail');
+
+/** Type override */
+export type Policies = t.TypeOf<typeof Policies>;
+
+/** the codec of a route enabled in an AI integration */
+export type EnabledRouteC = t.TypeC<{
+  /** the name of the enabled route */
+  routeName: t.Mixed;
+  /** the enabled policies */
+  enabledPolicies: t.ArrayC<t.LiteralC<Policies>>;
+}>;
+
+/** the codec of an AI Integration */
+export type AIIntegrationC = t.TypeC<{
+  /** the routes enabled in the AI integration */
+  enabledRoutes: t.ArrayC<EnabledRouteC>;
+}>;
```

Comment:
Just wondering -- why not use the same `t.TypeOf<..>` trick for these types too?

Reply:
We could! Will update it.